src/staking.rs

```diff
@@ -1572,176 +1615,169 @@ mod test {
             )
             .unwrap();
 
             // wait for unbonding period (60 seconds in default config)
-            test_env.block.time = test_env.block.time.plus_seconds(60);
+            env.block.time = env.block.time.plus_seconds(60);
 
             // need to manually cause queue to get processed
-            test_env
-                .router
+            env.router
                 .staking
-                .process_queue(
-                    &test_env.api,
-                    &mut test_env.store,
-                    &test_env.router,
-                    &test_env.block,
-                )
+                .process_queue(&env.api, &mut env.store, &env.router, &env.block)
                 .unwrap();
 
             // check bank balance
-            assert_balances(&test_env, vec![(delegator1.clone(), 1000)]);
+            assert_balances(&env, vec![(delegator_addr_1.clone(), 1000)]);
         }
 
         #[test]
         fn can_set_withdraw_address() {
-            let (mut test_env, validator) =
-                TestEnv::wrap(setup_test_env(Decimal::percent(10), Decimal::percent(10)));
+            let mut env = TestEnv::wrap(setup_test_env(10, 10));
 
-            let delegator = test_env.api.addr_make("delegator");
-            let reward_receiver = test_env.api.addr_make("rewardreceiver");
+            let validator_addr_1 = env.validator_addr_1();
+            let delegator_addr_1 = env.delegator_addr_1();
+            let reward_receiver_addr = env.user_addr_1();
 
-            test_env
-                .router
+            env.router
                 .bank
-                .init_balance(&mut test_env.store, &delegator, coins(100, "TOKEN"))
+                .init_balance(&mut env.store, &delegator_addr_1, coins(100, BONDED_DENOM))
                 .unwrap();
 
             // Stake 100 tokens to the validator.
             execute_stake(
-                &mut test_env,
-                delegator.clone(),
+                &mut env,
+                delegator_addr_1.clone(),
                 StakingMsg::Delegate {
-                    validator: validator.to_string(),
-                    amount: coin(100, "TOKEN"),
+                    validator: validator_addr_1.to_string(),
+                    amount: coin(100, BONDED_DENOM),
                 },
             )
             .unwrap();
 
             // Change rewards receiver.
             execute_distr(
-                &mut test_env,
-                delegator.clone(),
+                &mut env,
+                delegator_addr_1.clone(),
                 DistributionMsg::SetWithdrawAddress {
-                    address: reward_receiver.to_string(),
+                    address: reward_receiver_addr.to_string(),
                 },
             )
             .unwrap();
 
             // A year passes.
-            test_env.block.time = test_env.block.time.plus_seconds(60 * 60 * 24 * 365);
+            env.block.time = env.block.time.plus_seconds(60 * 60 * 24 * 365);
```

Comment:
Here you still manually calculate a year in seconds even though you defined a constant earlier 

(same for a bunch of other places, I'm on my phone so seeing all of them is a bit hard :( )

Reply:
Nice catch! Replaced remaining year calculations with a constant. Thanks for finding this!